protobuf/server_message.proto

```diff
@@ -39,10 +39,18 @@ message ServerMessage {
     bool pause = 1;
   }
 
+  message QueryRunPostgresSetting {
+    string name = 1;
+    string value = 2;
+  }
+
   message QueryRun {
     int64 id = 1;
     QueryRunType type = 2;
     string database_name = 3;
     string query_text = 4;
+    repeated NullString query_parameters = 5;
+    repeated string query_parameter_types = 6;
+    repeated QueryRunPostgresSetting postgres_settings = 7;
```

Comment:
On the app side we're calling these planner settings, not Postgres settings. Are we planning to extend this beyond planner settings to the less-specific name would make sense in the future?

Reply:
Uhmm good point. I'm happy to also update the app side to be more generic, and use postgres settings. If we do so, we can potentially specify any other things like overwriting `statement_timeout`.

Reply:
Hmm hmm. I think the main purpose here is to use it for anything that could affect the planner, which notably could include things like `work_mem` (though that is 50/50, since it partially affects execution, partially planning, depending on the node type). I'd argue though that `statement_timeout` is best managed separately (i.e. explicitly), since it has broader implications.

I'm fine either way, but we should be consistent. I think its probably most logical if I rename this here to `planner_settings` for now (I'll do that tomorrow unless I hear reasons not to).

Reply:
A reason to stick with `postgres_settings` is to have less code when in the future query runs can support more maintenance tasks (vacuum, reindex, create index, drop index), where it might make sense to set things like `work_mem`.

Reply:
Yeah, that's fair - the data structure here can be a bit more agnostic, and you're right that we could have utility commands that could benefit from having certain settings set. In that light I'm also okay with keeping this as-is. 

@keiko713 thoughts?

Reply:
Yeah, I agree that `postgres_settings` is a better naming here.
I think we can still debate a bit about if we should also update the wording we use in the app side or not (e.g. query run will use postgres_settings, but maybe explain_query can still use planner_settings as a column name, then generate postgres_settings based on planner_settings).

Reply:
Unresolving this conversation: it looks like the [first comment](https://github.com/pganalyze/collector/pull/655#discussion_r1896868627) got missed